### PR TITLE
Avoid dirty state and multiple request when opening empty smart content

### PR DESCRIPTION
**Pull request by danrot**
| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | ---
| Related issues/PRs | ---
| License | MIT
| Documentation PR | ---

#### What's in this PR?

This PR avoid settings the default values of the smart content after the form has been initialized.

#### Why?

Because if you've opened e.g. a new page, the save button was immediately active, without the content manager editing anything. And there were double requests on smart contents, because their value was changed afterwards.

### CODE REVIEW

File: src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/SmartContent.js [hunk 1]
```diff
@@ -25,49 +25,114 @@ const filterCriteriaDefaults = {
...
-                    value: excludeDuplicates = false,
+                present_as: {
+                    value: schemaPresentations = [],
                 } = {},
+            } = {},
+        } = this.props;
+
+        if (!Array.isArray(schemaPresentations)) {
+            throw new Error(
+                'The "present_as" schemaOption must be a string, but received ' + typeof schemaPresentations + '!'
```
Comment (nnatter) [inline src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/SmartContent.js]: i guess this should be `must be an array`

File: src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/SmartContent.js [hunk 2]
```diff
@@ -90,11 +155,11 @@ class SmartContent extends React.Component<Props> {
         this.filterCriteriaChangeDisposer();
     }
 
-    handleFilterCriteriaChange = () => {
+    handleFilterCriteriaChange = (filterCriteria: ?FilterCriteria) => {
         const {onChange, onFinish, value} = this.props;
 
         const currentValue = {...filterCriteriaDefaults, ...toJS(value)};
-        const newValue = {...filterCriteriaDefaults, ...toJS(this.smartContentStore.filterCriteria)};
+        const newValue = {...filterCriteriaDefaults, ...toJS(filterCriteria)};
```
Comment (nnatter) [inline src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/SmartContent.js]: is the `filterCriteriaDefaults` constanct still needed?

File: src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/SmartContent.js (hunk 1 above)
Comment (danrot) [reply src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/SmartContent.js]: right :slightly_smiling_face:

File: src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/SmartContent.js (hunk 2 above)
Comment (danrot) [reply src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/SmartContent.js]: Had it already delete, but then I had to revert a change :sweat_smile: Gonna fix it.

**Pull request merged**